src/olympia/landfill/serializers.py

```diff
@@ -241,3 +284,6 @@ def create_installable_addon(self):
             # And let's create a new version for that upload.
             create_version_for_upload(
                 upload.addon, upload, amo.RELEASE_CHANNEL_LISTED)
+
+            # Change status to public
+            addon.update(status=amo.STATUS_PUBLIC)
```

Comment:
It wasn't already public? You shouldn't be able to see the add-on if it's not unless we're logging in with some user?

Reply:
No, you had to navigate to the URL to find the addon.